src/box/decimal.h

```diff
@@ -353,7 +353,7 @@ box_decimal_minus(box_decimal_t *res, const box_decimal_t *dec);
  * @param lhs left hand side operand
  * @param rhs right hand side operand
  * @return NULL on an error (an overflow for example)
- * @return decimal result (@res)
```

Comment:
It means that the return value *is* the first parameter, `res`. I would replace it with `(@a res)`. The same for the functions below.

Reply:
replaced `@res` with `@a res` everywhere